plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -46,12 +46,10 @@ import {
 } from '@opentelemetry/semantic-conventions';
 import { VERSION } from './version';
 
-const PG_POOL_COMPONENT = 'pg-pool';
-
 export class PgInstrumentation extends InstrumentationBase {
   static readonly COMPONENT = 'pg';
```

Comment:
optional: I think it's a good time to get rid of this `COMPONENT` var which, after your PR, will only be used in one place for log.
If you have time and energy, please fell free to remove it as well :)